mitodl/mit-open
# randomize the hero image

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/5267

### Description (What does it do?)
This PR adds some new hero images and sets them up to be randomized each time you load the home page.

### Screenshots (if appropriate):
![image](https://github.com/user-attachments/assets/c586d544-96fc-40da-b6cc-9ce61395d324)

### How can this be tested?
 - Spin up this branch of `mit-learn`
 - Load the home page
 - Refresh several times and verify that the images are randomized
